### ISSUE TITLE
Index canister is missing in `@dfinity/ledger-icp`

### ISSUE THREAD

**Issue by seniorjoinu** [dfinity/ic-js#460]
Original source code:
```typescript
// index.ts

export { AccountIdentifier, SubAccount } from "./account_identifier";
export * from "./errors/ledger.errors";
export { IndexCanister } from "./index.canister";
export { LedgerCanister } from "./ledger.canister";
export type * from "./types/common";
export * from "./types/ledger.options";
export * from "./utils/account_identifier.utils";
export * from "./utils/accounts.utils";
```

Compiled code:
```typescript
// index.d.ts

export { AccountIdentifier, SubAccount } from "./account_identifier";
export * from "./errors/ledger.errors";

// <------------ no export { IndexCanister } from "./index.canister"; ---------- ???

export { LedgerCanister } from "./ledger.canister";
export * from "./types/ledger.options";
export * from "./utils/account_identifier.utils";
export * from "./utils/accounts.utils";
```

**Comment by peterpeterparker** [comment]
The support for the ICP index canister is not yet released in `@dfinity/ledger-icp`.

I actually just merged the PR that initiates the code and provides the first function this morning in #459. 

It will be released in the next version.

That's why I'm closing this issue, but if I missed something, please ping me, and I'll reopen it.

**Comment by peterpeterparker** [comment]
@seniorjoinu a new version of the `@dfinity/ledger-icp` has just been released (v1.1.0
) which now exposes this new `IndexCanister`.
